🐛(edxapp) fix jobs image name when using custom theme

rmoch opened the pull request:
## Purpose

When using a custom theme, `edxapp` jobs containers are supposed to run using a non-existing image since the theme version is included in the image name reference.

## Proposal

- [x] use the `image_name` macro in `edxapp` jobs

jmaupetit commented on apps/edxapp/templates/services/cms/_bc_base.yml.j2:
```diff
@@ -18,7 +18,7 @@ metadata:
   #
   # 1. https://docs.okd.io/latest/dev_guide/builds/triggering_builds.html#config-change-triggers
-  name: "{{ image_name(service_variant, edxapp_image_tag, edxapp_theme_tag) | replace(':', '-') }}"
+  name: "{{ internal_docker_registry }}/{{ project_name }}/{{ image_name(service_variant, edxapp_image_tag, edxapp_theme_tag) }}"
```
This should not be changed.

jmaupetit commented on apps/edxapp/templates/services/cms/job_01_internationalization.yml.j2:
```diff
@@ -27,7 +27,7 @@ spec:
           # We point to a local registry image build for this "live" image (see
           # ImageStream and BuildConfig templates)
-          image: "{{ internal_docker_registry }}/{{ project_name }}/edxapp-cms:{{ edxapp_image_tag }}-live"
+          image: "{{ internal_docker_registry }}/{{ project_name }}/{{ image_name(service_variant, edxapp_image_tag, edxapp_theme_tag) }}"
```
`service_variant` variable is not defined here, should be:

```suggestion
          image: "{{ internal_docker_registry }}/{{ project_name }}/{{ image_name('cms', edxapp_image_tag, edxapp_theme_tag) }}"
```

jmaupetit replied:
The same applies for all other job templates modifications.

jmaupetit committed:
🐛(edxapp) fix jobs image name when using custom theme

When using a custom theme, edxapp jobs containers are supposed to run
using a non-existing image since the theme version is included in the
image name reference.
